### ISSUE TITLE
Germany/Switzerland/Austria is missing as a region

### ISSUE THREAD

**Issue by warum7** [hadynz/obsidian-kindle-plugin#99]
By adding the following Object a new region could be added
```ts
germany: {
    name: 'Germany/Switzerland/Austria',
    hostname: 'amazon.de',
    kindleReaderUrl: 'https://lesen.amazon.de',
    notebookUrl: 'https://lesen.amazon.de/notebook',
  },
```

**Comment by hadynz** [comment]
You were on point. Thanks for digging up the exact syntax to add a new region. Feel free to make a PR next time :) 

Will add this shortly in the next upcoming release.

**Comment by hadynz** [comment]
This was released in [v1.1.1][1]. Let me know if it works for you.

[1]: https://github.com/hadynz/obsidian-kindle-plugin/releases/tag/1.1.1

**Comment by warum7** [comment]
> This was released in [v1.1.1](https://github.com/hadynz/obsidian-kindle-plugin/releases/tag/1.1.1). Let me know if it works for you.

Working perfectly fine! Thank you very much for the blazing fast implementation. Next time I will do a PR :)
Love the plugin by the way. Implementing it in my workflow:
Kindle -> Sync -> Important stuff to anki via the anki sync plugin.

**Comment by hadynz** [comment]
I vaguely remember hearing about Anki but I've never used it before. Will try it sometime as spaced repetition can be very effective. 

Closing this issue since it has been resolved.
